Gemfile

```diff
@@ -30,7 +30,7 @@ end
 if puppetversion = ENV['PUPPET_GEM_VERSION']
   gem 'puppet', puppetversion, :require => false
 else
-  gem 'puppet', :require => false
+  gem 'puppet', '< 4.0.0', :require => false
```

Comment:
puppetlabs_spec_helper-0.10.2 seems to be incompatible with puppet 4.0. I get this error:

```
/usr/bin/ruby1.9.1 -S rspec spec/classes/sensu_api_spec.rb spec/classes/sensu_client_spec.rb spec/classes/sensu_init_spec.rb spec/classes/sensu_package_spec.rb spec/classes/sensu_rabbitmq_spec.rb spec/classes/sensu_redis_spec.rb spec/classes/sensu_server_spec.rb spec/defines/sensu_check_spec.rb spec/defines/sensu_extension_spec.rb spec/defines/sensu_filter_spec.rb spec/defines/sensu_handler_spec.rb spec/defines/sensu_plugin_spec.rb spec/defines/sensu_subscription_spec.rb spec/unit/puppet_x_sensu_totype_spec.rb spec/unit/sensu_config_spec.rb --color
/home/glopes/.gems/gems/puppetlabs_spec_helper-0.10.2/lib/puppetlabs_spec_helper/module_spec_helper.rb:23:in `block in <top (required)>': undefined method `environmentpath=' for #<RSpec::Core::Configuration:0x000000012d2ae8> (NoMethodError)
	from /var/lib/gems/1.9.1/gems/rspec-core-2.99.2/lib/rspec/core.rb:154:in `configure'
	from /home/glopes/.gems/gems/puppetlabs_spec_helper-0.10.2/lib/puppetlabs_spec_helper/module_spec_helper.rb:22:in `<top (required)>'
	from /home/glopes/repos/sensu-puppet/spec/spec_helper.rb:2:in `require'
	from /home/glopes/repos/sensu-puppet/spec/spec_helper.rb:2:in `<top (required)>'
	from /home/glopes/repos/sensu-puppet/spec/classes/sensu_api_spec.rb:1:in `require'
	from /home/glopes/repos/sensu-puppet/spec/classes/sensu_api_spec.rb:1:in `<top (required)>'
	from /var/lib/gems/1.9.1/gems/rspec-core-2.99.2/lib/rspec/core/configuration.rb:1065:in `load'
	from /var/lib/gems/1.9.1/gems/rspec-core-2.99.2/lib/rspec/core/configuration.rb:1065:in `block in load_spec_files'
	from /var/lib/gems/1.9.1/gems/rspec-core-2.99.2/lib/rspec/core/configuration.rb:1065:in `each'
	from /var/lib/gems/1.9.1/gems/rspec-core-2.99.2/lib/rspec/core/configuration.rb:1065:in `load_spec_files'
	from /var/lib/gems/1.9.1/gems/rspec-core-2.99.2/lib/rspec/core/command_line.rb:18:in `run'
	from /var/lib/gems/1.9.1/gems/rspec-core-2.99.2/lib/rspec/core/runner.rb:103:in `run'
	from /var/lib/gems/1.9.1/gems/rspec-core-2.99.2/lib/rspec/core/runner.rb:17:in `block in autorun'
```